uds/uds_config_tool/UdsConfigTool.py

```diff
@@ -13,101 +13,69 @@
-)
-from uds.uds_config_tool.FunctionCreation.ReadDTCMethodFactory import (
-    ReadDTCMethodFactory,
-)
-from uds.uds_config_tool.FunctionCreation.RequestDownloadMethodFactory import (
-    RequestDownloadMethodFactory,
-)
-from uds.uds_config_tool.FunctionCreation.RequestUploadMethodFactory import (
-    RequestUploadMethodFactory,
-)
-from uds.uds_config_tool.FunctionCreation.RoutineControlMethodFactory import (
-    RoutineControlMethodFactory,
-)
-from uds.uds_config_tool.FunctionCreation.SecurityAccessMethodFactory import (
-    SecurityAccessMethodFactory,
-)
-from uds.uds_config_tool.FunctionCreation.TesterPresentMethodFactory import (
-    TesterPresentMethodFactory,
-)
-from uds.uds_config_tool.FunctionCreation.TransferDataMethodFactory import (
-    TransferDataMethodFactory,
-)
-from uds.uds_config_tool.FunctionCreation.TransferExitMethodFactory import (
-    TransferExitMethodFactory,
-)
-from uds.uds_config_tool.FunctionCreation.WriteDataByIdentifierMethodFactory import (
-    WriteDataByIdentifierMethodFactory,
-)
-from uds.uds_config_tool.ISOStandard.ISOStandard import (
-    IsoInputOutputControlOptionRecord,
-)
-from uds.uds_config_tool.ISOStandard.ISOStandard import (
-    IsoReadDTCStatusMask as Mask,
-)
-from uds.uds_config_tool.ISOStandard.ISOStandard import (
-    IsoReadDTCSubfunction,
-    IsoRoutineControlType,
-    IsoServices,
-)
-from uds.uds_config_tool.SupportedServices.ClearDTCContainer import (
-    ClearDTCContainer,
-)
-from uds.uds_config_tool.SupportedServices.DiagnosticSessionControlContainer import (
-    DiagnosticSessionControlContainer,
-)
-from uds.uds_config_tool.SupportedServices.ECUResetContainer import (
-    ECUResetContainer,
-)
-from uds.uds_config_tool.SupportedServices.InputOutputControlContainer import (
-    InputOutputControlContainer,
-)
-from uds.uds_config_tool.SupportedServices.ReadDataByIdentifierContainer import (
-    ReadDataByIdentifierContainer,
-)
-from uds.uds_config_tool.SupportedServices.ReadDTCContainer import (
-    ReadDTCContainer,
-)
-from uds.uds_config_tool.SupportedServices.RequestDownloadContainer import (
-    RequestDownloadContainer,
-)
-from uds.uds_config_tool.SupportedServices.RequestUploadContainer import (
-    RequestUploadContainer,
-)
-from uds.uds_config_tool.SupportedServices.RoutineControlContainer import (
-    RoutineControlContainer,
-)
-from uds.uds_config_tool.SupportedServices.SecurityAccessContainer import (
-    SecurityAccessContainer,
-)
-from uds.uds_config_tool.SupportedServices.TesterPresentContainer import (
-    TesterPresentContainer,
-)
-from uds.uds_config_tool.SupportedServices.TransferDataContainer import (
-    TransferDataContainer,
-)
-from uds.uds_config_tool.SupportedServices.TransferExitContainer import (
-    TransferExitContainer,
-)
-from uds.uds_config_tool.SupportedServices.WriteDataByIdentifierContainer import (
-    WriteDataByIdentifierContainer,
-)
+from uds.uds_config_tool.FunctionCreation.ClearDTCMethodFactory import \
```

Comment:
Why are all the imports incorrectly formatted?

Reply:
Probably conflicting settings between isort and black, should be fixed after running black again.